app/javascript/components/KeyWordSearch.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import Button from 'react-bootstrap/lib/Button';
+import InputGroup from 'react-bootstrap/lib/InputGroup';
+import Form from 'react-bootstrap/lib/Form';
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+
+class KeyWordSearch extends React.Component{
+  constructor(props){
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.state = {
+      searchTerms: undefined,
+    };
+  };
+
+  handleSubmit(e){
+    // Prevent full page reload
+    e.preventDefault();
+
+    const searchTerm = e.target.elements.searchText.value.trim();
```

Comment:
does this logic correctly handle the case where a user sumbits a keyword search, then clears the field and hits enter again to see unfiltered results?

Reply:
That logic would be controlled by the parent class. In the event where there's keyword search is empty, it should show the original set of studies AKA the default state.